handler/Dockerfile

```diff
@@ -72,8 +62,9 @@ RUN git clone --branch development https://github.com/rordenlab/dcm2niix.git \
 RUN cd && git clone https://github.com/bids-standard/bids-specification
 
 #install ROBEX
-ADD https://www.nitrc.org/frs/download.php/5994/ROBEXv12.linux64.tar.gz//?i_agree=1&download_now=1 /
-RUN tar -xzf /ROBEXv12.linux64.tar.gz
+RUN mkdir /ROBEX && \
```

Comment:
@anibalsolon, this is causing an error when trying to use `quickshear` for anatomical defacing.

```
failed to run deface -- code:1
+ defaced=./quick_nifti_test/time-20200122142947-sn-2-name-anat-T1w.nii.gz.defaced.nii.gz
+ echo '--------------- defacing(quickshear) [0] ./quick_nifti_test/time-20200122142947-sn-2-name-anat-T1w.nii.gz to ./quick_nifti_test/time-20200122142947-sn-2-name-anat-T1w.nii.gz.defaced.nii.gz ----------------'
+ case $method in
+ runROBEX.sh ./quick_nifti_test/time-20200122142947-sn-2-name-anat-T1w.nii.gz ./quick_nifti_test/time-20200122142947-sn-2-name-anat-T1w.nii.gz.mask.nii.gz
environment: line 10: runROBEX.sh: command not found

real	0m0.000s
user	0m0.000s
sys	0m0.000s
```